app/frontend/src/assets/scss/style.scss

```diff
@@ -361,6 +361,19 @@ a,
     }
   }
 
+  // Removing default (optional) flag on Custom Day Component
+  label[for$="simpleday"]:after {
+    content: "";
+  }
+
+  // Adding (optional) flag to Checkbox cases
+  :is(.form-check.checkbox) label {
+      &:after {
+        content: " (optional)";
+        color: inherit;
+      }
+  }
```

Comment:
here you are overriding line 353. so you could just change line 353 to:
` label { ` and you wont need this

Reply:
I was actually writing this in order to override line 353.

1. Line 365 is used to target the label for the Day Component
2. Line 370 is targeting the .form-check.checkbox class to remove the optional flag by overriding Line 353

We can talk about scss refactor on Friday

Reply:
After clarification, this has been changed and fixed.